### PR TITLE
Use newer go version

### PR DESCRIPTION
**Pull request by kkohbrok**

### CODE REVIEW

File: .github/workflows/interop.yml [hunk 1]
```diff
@@ -80,12 +80,17 @@ jobs:
           # TODO(#1366)
           cp interop_client/docker/test-runner/main.go.patch mls-implementations/interop/test-runner
           cd mls-implementations/interop
+          go mod edit -go=1.18
```
Comment (franziskuskiefer) [inline .github/workflows/interop.yml:83]: 1.18 is pretty old (2022). Maybe updating to something recent solves the issue?

**Review by franziskuskiefer — COMMENTED** [review]
I'd also be ok disabling this job for now. Or at least not requiring it.
Something needs to get fixed properly here.

File: .github/workflows/interop.yml (hunk 1 above)
Comment (raphaelrobert) [reply .github/workflows/interop.yml:83]: 1.22 is the most recent and worked for me locally

**Comment by kkohbrok** [comment]
#1594 did the trick.

**Pull request closed without merging**